docs/api/bluetooth/Gap.md

```diff
@@ -52,13 +52,15 @@ You may set preferred PHYs (separately for RX and TX) using `setPreferredPhys()`
 
 You may query the currently used PHY using `readPhy()`, which returns the result through a call to the registered event handler. You may register the handler with `setEventHandler()`. The events inform about the currently used PHY and of any changes to PHYs, which the controller or the peer may trigger autonomously.
 
-#### Data Length (over-the-air MTU)
+#### Data length (over-the-air MTU)
 
-In addition to modulation schemes, MTU (Maximum Transmission Unit) size also strongly affects throughput. Newer controllers will allow you to negotiate bigger MTUs. Since each packet contains overhead bigger packets maximise throughput.
+In addition to modulation schemes, Maximum Transmission Unit (MTU) size also strongly affects throughput. Newer controllers allow you to negotiate bigger MTUs. Because each packet contains overhead, bigger packets maximize throughput.
```

Comment:
Why is Because superior to Since in this case? Maybe I can try to choose the right one in the future.

Reply:
Great question. "Since" shows time passed (for example, "We've been open since 1952"). "Because" shows cause and effect.

Reply:
Not trying to open a debate here (you know better than us @AnotherButler) but there is other usages of _since_ (see https://dictionary.cambridge.org/us/grammar/british-grammar/as-because-or-since). Should we avoid these forms in our documentation.

Reply:
Please do. We try to be consistent in our documentation, and [our internal style guide](https://docs.internal.mbed.com/docs/master/style/usage-and-spelling-of-words-you-usually-get-wrong.html) states "Because implies causality; since implies the passing of time. Examples: The program ran because I did it right. Arm has been in business since the 1990s." and "But also note that using "as" when you mean "because" can confuse non-native speakers, so it's best to avoid it." Please don't hesitate to look up words in our style guide if you ever have questions. We can add to it if anything's missing. 

We're also happy to answer questions like this one :)